src/scripts/components/CRCMaker.js

```diff
@@ -254,14 +264,16 @@ export default class CRCMaker extends Component {
                   </Dialog>
                 }
 
+                <button onClick={this.toggleImport}>Import</button>
+
```

Comment:
I'd rather the import dialog be around here too, since the export dialog is here.

Reply:
I don't think that's possible because header is hidden on the initial load